### DISCUSSION TITLE
The Copilot page’s input box is blocked by the iPadOS IME indicator.

### DISCUSSION THREAD

**Discussion by RA1NO3O** [community/community#125548] [Mobile]
### Select Topic Area

Product Feedback

### Body

This bug is only appears when iPad using hardware keyboard.
![IMG_1312](https://github.com/community/community/assets/25654559/85473770-591f-4fdc-8b5d-faed6f3de77c)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.162.0 (130645053)
Device: iPad16,3 (iOS 17.5.1)
</details>

**Comment by taki-on** [comment]
@RA1NO3O Thank you for reporting the issue. It is currently known to us, and we are figuring out a solution. Please stay tuned!
